python/tvm/target/tag.py

```diff
@@ -57,14 +60,14 @@ def register_tag(name: str, config: Dict[str, Any], override: bool = False) -> T
             "arch": "sm_61",
         })
     """
-    return _ffi_api.TargetTagAddTag(name, config, override)
+    if hasattr(_ffi_api, "TargetTagAddTag"):
+        return _ffi_api.TargetTagAddTag(name, config, override)
+    return None
 
 
-# We do a round of tag listing on loading time to verify all tag information
-if hasattr(_ffi_api, "TargetTagAddTag"):
-    list_tags()
+list_tags()
```

Comment:
Per offline discussion, the purpose of making this call is to check the tag correctness. Based on this, it would be better to make an explicit description in `list_tags()`, saying that this function also performs correctness checking.

Reply:
Added some description in code comments.